src/main/java/FlightsReporter.groovy

```diff
@@ -0,0 +1,257 @@
+import java.time.LocalDateTime
+import java.time.temporal.ChronoUnit
+
+class FlightsReporter {
+    final static int[] CAPACITIES = new int[] {80, 120, 150, 180, 240}
+
+    static List<Flight> getAllFlightsTo(List<Flight> flights, Airports destination) {
+        flights.findAll { (it.destination == destination) }
+    }
+
+    static List<Flight> getAllFlightsFrom(List<Flight> flights, Airports from) {
+        flights.findAll {it.from == from }
```

Comment:
Use the auto-formatter . The { are really close to the code and it isn't easy to read